docs/source/getting-started.rst

```diff
@@ -109,15 +109,13 @@ Now that you've gotten the lay of the land, let's dig in!
 
 .. _gs-maven-setup:
 
-Setting Up Maven
+Setting Up Using Maven
 ================
```

Comment:
Could you please fix the length of the line? Sphinx should give you related warnings.